platforms/corda-ent/configuration/roles/helm_component/templates/node.tpl

```diff
@@ -28,48 +28,48 @@ spec:
     networkServices:
       doormanURL: {{ doorman_url }}
       networkMapURL: {{ networkmap_url }}
-      idmanDomain: {{ idman_domain }}
-      networkMapDomain: {{ networkmap_domain }}
+      idmanDomain: "{{ doorman_url.split(':')[1] | regex_replace('/', '') }}"
+      networkMapDomain: "{{ networkmap_url.split(':')[1] | regex_replace('/', '') }}"
     firewall:
-      enabled: {{ firewall_enabled }}
+      enabled: {{ peer.firewall.enabled }}
     cenmServices:  
```

Comment:
cenmservices to be merged with networkservices

Reply:
done